core/templates/domain/exploration/solution.model.ts

```diff
@@ -155,10 +152,7 @@ export class Solution {
   }
 }
 
-@Injectable({
-  providedIn: 'root'
-})
-export class SolutionObjectFactory {
+export class Solutions {
```

Comment:
Previously it was a singular (solution), now it is plural (solutions), why is that?

Reply:
In ```solution.model.ts```, Solution was already present with different definition(check line 64-153) and (check line 155-178) for definition of Solutions.

Reply:
I think, conceptually, it has to be Solution because there can only be one solution to a card. @vojtechjelinek / @srijanreddy98 can you advice on this?

Reply:
If it has to be ```Solution```, then what about the definition of previously existing ```Solution```(line 64-153)?

Reply:
Ah I see what is the problem, as per the [instructions](https://github.com/oppia/oppia/wiki/Angular-Migration#3-move-functions-from-objectfactory-class) you should not create a new class but rather move some of the functions from the `SolutionObjectFactory` to `Solution` and then remove `SolutionObjectFactory`.